lib/beetle_etl/naming.rb

```diff
@@ -1,14 +1,25 @@
 require 'digest'
+require 'active_support/inflector'
 
 module BeetleETL
   module Naming
-
-    extend self
+    module_function
 
     def stage_table_name(external_source, table_name)
       digest = Digest::MD5.hexdigest(table_name.to_s)
-      "#{external_source.to_s}-#{table_name.to_s}-#{digest}"[0, 63]
+      "#{external_source}-#{table_name}-#{digest}"[0, 63]
+    end
+
+    def mappings_table_name(table_name)
+      add_prefix(table_name, 'external_system_mappings')
     end
 
+    def mapped_foreign_key_column(table_name)
+      add_prefix(table_name, 'id')
+    end
+
+    def add_prefix(table_name, suffix)
```

Comment:
This one should be private.

Reply:
It does not work. I think because it's a module.

Reply:
https://stackoverflow.com/questions/318850/private-module-methods-in-ruby